src/protocols/secure_channel/CheckinMessage.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file implements the utility class for the Matter Checkin protocol.
+ */
+
+#include "CheckinMessage.h"
+#include <lib/core/CHIPCore.h>
+
+namespace chip {
+namespace Protocols {
+namespace SecureChannel {
+
+static const uint16_t mMaxPlainTextSize = CHIP_CHECK_IN_APP_DATA_MAX_SIZE + CHIP_CRYPTO_AEAD_NONCE_LENGTH_BYTES;
```

Comment:
Statics should probably start with "s".  And be constexpr.

Reply:
done